core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -153,14 +151,15 @@ object TestUtils extends Logging {
     enableDeleteTopic: Boolean = false,
     interBrokerSecurityProtocol: Option[SecurityProtocol] = None,
     trustStoreFile: Option[File] = None,
+    saslProperties: Option[Properties] = None,
```

Comment:
I'm wondering if this is really necessary. Could we instead add the sasl properties to the properties returned by this method via a utility method that added them only if necessary? It seems like we don't gain much by doing it this way and it adds one more parameter to a very large number of parameters already.